tools/exporter/README.md

```diff
@@ -12,66 +12,64 @@ This will install
 * iAuditor Exporter Tool
 * README files
 
-
 ## Initial Setup
-IMPORTANT: If you have used previous versions of the exporter tool, you should run the new version of the tool from the same folder you have run the tool from in the past.
+IMPORTANT: If you are using a version prior to 2.0.0 (see [here](https://github.com/SafetyCulture/safetyculture-sdk-python/releases)), you should run the new version of the tool from the same folder you ran the earlier version of the tool.
 Otherwise, the exporter will start exporting from the earliest available audits, rather than from where the last successful export left off. 
-The export tool reads and writes to a file named `last_successful.txt` to keep track of what has already been exported. You'll find this file anywhere that you have run the exporter tool before. 
+The export tool reads and writes to a file named `last_successful.txt` to keep track of what audits have already been exported. You'll find this file anywhere that you have run the exporter tool before. 
 Alternatively, you can move the `last_successful.txt` file if you prefer to export from a different location.
 
-If this is your first time using the exporter tool, follow these steps to get set up: 
+If this is your first time using the exporter tool, follow these steps to get set up after installation: 
 1. To automatically create a configuration file (which is needed to run the exporter tool), run:  
 ```
 iauditor_exporter --setup
 ```
 * You will be prompted for an iAuditor username and password which will be used to generate an API token. 
 Note that your username and password will not be saved, only used to generate the API token which is saved in the auto-generated configuration file.
 * A configuration file is necessary to run the Exporter script. The file will be named `config.yaml` and be placed in a folder named `iauditor_exports_folder` which will be created in your current directory. 
-2. Navigate into the `iauditor_exports_folder` folder just created:
+2. Navigate into the newly created folder `iauditor_exports_folder`:
 ```
-cd 'iauditor_exports_folder'
+cd iauditor_exports_folder
 ```
-3. To start exporting audits in PDF format, run the following command 
+3. To start exporting audits in PDF format, run the following command: 
 ```
 iauditor_exporter
 ```
 #### Windows Users
-The location of `iauditor_exporter.exe` must be included in the system PATH variable in order to execute from the command line without including the full path to the .exe file.  
+The location of `iauditor_exporter.exe` must be included in the system PATH variable in order to execute from the command line.  
```

Comment:
Is this really necessary?

Reply:
Perhaps not, I'll get Malik's opinion. He is going to test on his Windows system. Maybe most people will have the PATH variable setup correctly already